libsolidity/codegen/CompilerContext.h

```diff
@@ -121,6 +123,7 @@ class CompilerContext
 	);
 	/// Generates the code for missing low-level functions, i.e. calls the generators passed above.
 	void appendMissingLowLevelFunctions();
+	ABIFunctions& abiFunctions() { return m_abiFunctions; }
```

Comment:
Comment needed her. Also mention this is not constant because utils access this to append to it.

Reply:
What kind of comment would you suggest? Returns the abi functions module? Isn't that what you would expect even without a comment?

Reply:
I guess the only important part is why it is not constant.

Reply:
it is not constant because it returns a non-const ref and that is required because all functions in ABIFunctions are non-const. Do we really have to explain that? I mean you only use this function if you know what ABIFunctions is all about.